spec/game_spec.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+describe 'A game of bowling' do
+  let(:game) { Game.new }
+  let(:rolls) { 17.times { game.roll(0) } }
```

Comment:
Given that you are only replacing one line by one line here, it probably would make it more readable to just have the rolls in the tests.